### PR TITLE
[next-gen] Use new setup script

### PR DESCRIPTION
**Pull request by Jarrku**
> See JIRA https://skedify.atlassian.net/browse/SKED-

## TODO
- [x] Use stable package version when `frontend-mono` PR is merged.

### CODE REVIEW

File: react-ts/basic/src/App.tsx [hunk 1]
```diff
@@ -11,7 +11,7 @@ export const PLUGIN_SCRIPT_SRC =
 function Playground() {
   return (
     <div>
-      <PexipEngagePlugin version="1.0.0" />
```
Comment (BenSwennen) [inline react-ts/basic/src/App.tsx:14]: In the docs we mention that we recommend to always add a version
https://github.com/skedify/frontend-mono/pull/2730/files#diff-738ab28141ad7e8362963ad5d3e43a30c13415dc11ef6152446e03733b17a86fR71

Or has this been moved somewhere else?

**Comment by BenSwennen** [comment]
I get following warnings (polyfill + version) when running vanilla:

```
Polyfilling window.PexipEngage, this should not happen. Please make sure you're using the correct initialization script.
```
```
pexip-engage-plugin.js:1 [PEXIP-ENGAGE]: Failed to detect config version, assuming latest version.
Please add a version to your config to avoid breaking it in the future.
Current version: 1.0.0
```

Should we fix that?

---

For react example I get:

```
client-1e6460096ed2a.js:2 Prettier error:  Unexpected token, expected "," (2:8)
  1 | import {
> 2 |   type PexipEngagePluginContextProps,
    |        ^
  3 |   PexipEngagePluginProvider,
  4 |   PexipEngagePlugin,
  5 | } from "@pexip-engage-public/plugin-react";

```

**Comment by Jarrku** [comment]
Yes, I'll check

File: react-ts/basic/package.json [hunk 2]
```diff
@@ -10,20 +10,20 @@
     "preview": "vite preview"
   },
   "dependencies": {
-    "@pexip-engage-public/plugin-react": "^2.0.12",
+    "@pexip-engage-public/plugin-react": "2.0.17",
```
Comment (BenSwennen) [inline react-ts/basic/package.json]: Probably need `^` `"^2.0.17"`? @Jarrku

Comment (Jarrku) [reply react-ts/basic/package.json]: Can do but shouldnt matter

**Pull request merged**